CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
Damn I just realized I had like a dozen PRs and not one of them updated the changelog 😞